ProjectBank/Infrastructure.Tests/ProjectRepositoryTests.cs

```diff
@@ -266,35 +271,20 @@ public async Task ReadProjectsByStudentId_given_StudentId_returns_list_of_Projec
         Assert.Throws<System.ArgumentOutOfRangeException>(() => projects.ElementAt(3));
     }
 
-    [Fact]
-    public async Task UpdateProjectStatusById_given_ProjectId_changes_status()
-    {
-        await _projectRepository.CreateProjectAsync(new CreateProjectDTO { Title = "Algo", SupervisorId = "1", Description = "Very fun", Tags = new List<string> { "Sorting" } });
-
-        await _projectRepository.CloseProjectByIdAsync(1);
-
-        var expected = Status.Closed;
-        var actual = _context.Projects.Find(1).Status;
-
-        Assert.Equal(expected, actual);
-
-    }
-
     [Fact]
     public async Task AddUserToProject_given_ProjectId_and_StudentId_adds_user()
```

Comment:
If this also tests not found either the method name should reflect that or it should be a separate method